### DISCUSSION TITLE
NFT generated for challenge 1 (challenging but super fun)

### DISCUSSION THREAD

**Discussion by vishalsingh2972** [Cyfrin/foundry-full-course-cu#2261] [General]
I was successfully able to generate the sepolia NFT for challenge 1 
![image](https://github.com/user-attachments/assets/a36009dc-eadc-45ce-8c6a-3c2da067a793)

opensea link: https://testnets.opensea.io/assets/sepolia/0x76b50696b8effca6ee6da7f6471110f334536321/2883/

I was also successfully able to import the NFT into my metamask wallet but the image of the NFT was not visible in my metamask wallet even after trying multiple times
![image](https://github.com/user-attachments/assets/6a828517-58f0-4ad1-8a50-3b02298d2656)

So I skimmed through the discussion section and found this https://github.com/Cyfrin/foundry-full-course-cu/discussions/1764#discussioncomment-9729874 , so most probably its a metamask wallet issue and metamask is still trying to solve it. Hopefully will be able to see the image of the NFT in my metamask wallet soon. In case someone has another resolution to this please comment below.

**Comment by EngrPips** [comment]
Yeah, I assume it is a Metamask problem, and I wouldn't sweat over it.

**Comment by Totiruzi** [comment]
Please where do you find the instruction for the challenge or what to solve?
I just finishes Solidity 101 Section 1
I see NFTs at the end of the github content for the class and 2 links 
zkSync nad Sepolia.
They have not instructions or what to do?
Thanks )

**Reply by EngrPips** [reply]
Read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018

**Reply by Totiruzi** [reply]
Thanks a bunch )

**Reply by EngrPips** [reply]
You are welcome, Friend.
